lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -700,6 +700,7 @@ void ResourceMap::loadDefaultConfiguration()
   addAsString("Drawable-NoSpecifiedLabel", "");
   addAsUnsignedInteger("Drawable-DataThreshold", 2000);
   addAsUnsignedInteger("Drawable-DefaultPalettePhase", 12);
+  addAsString("Drawable-DefaultPaletteName", "Tableau");
```

Comment:
Please insert the new keys according to the rules: Bool first, then Scalar, then String, then UnsignedInteger, and sort the keys in alphabetical order within a given type

Reply:
Ok, I didn't notice there was an ordering. Done.